README.md

```diff
@@ -1,2 +1,90 @@
-# sim-ln
-Payment activity generator for the lightning network
+# Sim LN
+
+Sim LN is a simulation tool that can be used to generate realistic 
+payment activity on any lightning network topology. It is intentionally 
+environment-agnostic so that it can be used across many environments - 
+from integration tests to public signets. 
+
+This tool is intended to serve developers who are familiar with 
+lightning network development. It may be useful to you if you are:
+* A protocol developer looking to test proposals. 
+* An application developer load-testing your application.
+* A signet operator interested in a hands-off way to run an active node. 
+* A researcher generating synthetic data for a target topology.
+
+## Pre-Requisites
+Sim LN requires you to "bring your own network" to generate activity 
+on. You will need:
+* A [lightning network](#lightning-environments) connected with any 
+  topology of channels.
+* Access to execute commands on _at least_ one node in the network.
+
+This tool currently only supports LND nodes, but [will support](https://github.com/bitcoin-dev-project/sim-ln/issues/26)
+all major implementations in future versions.
+
+## Configuration
+The simulator is configured with two sets of information expressed in 
+json: 
+* `nodes`: a set of nodes that you have permission to execute 
+  commands on.
+* `activity`: a description of the payment activity that you would 
+  like to generate.
+
+The example config below sets up the following simulation:
+* Connect to a single node `Alice` to generate activity.
+* Dispatch 100 msat payments from `Alice` to `Carol` every 10 seconds.
+* Dispatch 100000 msat payments from `Alice` to `Eve` every 5 minutes.
+```
+{
+    "nodes": [ 
+    {
+        // Alice
+        "id": "0257956239efc55dd6be91eff40c47749314ccf79cb15f79e30ca12f8622b6de9e",
+        "address": "https://localhost:10011",
+        "macaroon": "/path/admin.macaroon",
+        "cert": "/path/tls.cert"
+    }],
+    "activity": [ 
+    {
+        // Alice -> Carol
+        "source": "0257956239efc55dd6be91eff40c47749314ccf79cb15f79e30ca12f8622b6de9e",
+        "destination": "02d804ad31429c8cc29e20ec43b4129553eb97623801e534ab5a66cdcd2149dbed",
+        "interval_secs": 10,
+        "amount_msat": 100
+    },
+	{
+        // Alice -> Eve
+        "source": "0257956239efc55dd6be91eff40c47749314ccf79cb15f79e30ca12f8622b6de9e",
+        "destination": "03232e245059a2e7f6e32d6c4bca97fc4cda935c553ea3693adb3265a19050c3bf",
+        "interval_secs": 300,
+        "amount_msat": 100000
+    }]
+}
+```
+
+## Getting Started
+To install from source, you will need the Rust compiler installed on 
```

Comment:
I'd move this to `Pre-Requisites` and keep it simple:

You will need:
[...]
- The rust compiler installed. Follow [this](https://www.rust-lang.org/tools/install) otherwise.